SecurityService.OpenIdConnect.IntegrationTests/Common/Hooks.cs

```diff
@@ -77,33 +77,30 @@
                 experimentalFlags.Add("same-site-by-default-cookies@2");
                 experimentalFlags.Add("cookies-without-same-site-must-be-secure@2");
                 options.AddLocalStatePreference("browser.enabled_labs_experiments", experimentalFlags);
+
                 this.WebDriver = new ChromeDriver(options);
             }
 
             if (browser == "Firefox")
             {
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6384137)